src/eulerevaluator.cpp

```diff
@@ -38,6 +38,8 @@ void EulerEvaluator::InitializeEquationParts() {
 	}
 }
 
+//! \file
```

Comment:
What's this for?

Reply:
a doxygen marker for it to scan the .cpp file, so i can include the snippet

Reply:
That's pretty neat